flare_portal/experiments/models/modules.py

```diff
@@ -372,10 +372,10 @@ class AffectiveRatingModule(BaseModule):
         ("gsd", "GSD"),
     )
 
-    stimulus = models.CharField(max_length=3, choices=STIMULI, default=STIMULI.csa)
     question = models.CharField(
         max_length=255, default="Have you seen this image before?"
```

Comment:
Just checking if we still need the `STIMULI` choices. If not we should remove it.

Reply:
Removed.